### ISSUE TITLE
omnisharp-mono fails to install on OSX

### ISSUE THREAD

**Issue by bfops** [OmniSharp/homebrew-omnisharp-roslyn#8]
```
$ brew install omnisharp-mono
==> Installing omnisharp-mono from omnisharp/omnisharp-roslyn
==> Downloading https://github.com/OmniSharp/omnisharp-roslyn/releases/download/v1.19.0/omnisharp-mono.tar.gz
Already downloaded: /Users/zeke/Library/Caches/Homebrew/omnisharp-mono-1.19.0.tar.gz
Error: Calling <<-EOS.undent is disabled!
Use <<~EOS instead.
/usr/local/Homebrew/Library/Taps/omnisharp/homebrew-omnisharp-roslyn/omnisharp-mono.rb:18:in `install'
Please report this to the omnisharp/omnisharp-roslyn tap!
Or, even better, submit a PR to fix it!
```

**Comment by bfops** [comment]
Oh looks like #7 fixes this

**Comment by filipw** [comment]
fixed with #12
